actix-server/Cargo.toml

```diff
@@ -17,18 +17,17 @@ edition = "2018"
 name = "actix_server"
 path = "src/lib.rs"
 
-[features]
-default = []
-
 [dependencies]
 actix-rt = { version = "2.0.0", default-features = false }
 actix-service = "2.0.0-beta.5"
 actix-utils = "3.0.0-beta.4"
 
 futures-core = { version = "0.3.7", default-features = false, features = ["alloc"] }
+futures-util = { version = "0.3.7", default-features = false }
 log = "0.4"
-mio = { version = "0.7.6", features = ["os-poll", "net"] }
+mio = { version = "0.7.11", features = ["os-poll", "net", "os-util"] }
 num_cpus = "1.13"
+rtrb = ""
```

Comment:
version

Reply:
Still not decided what crate to use. There is no obvious choice for now.

Reply:
it's all internal so wouldn't worry, the interface over the channel impl seems reasonable

Reply:
It's mostly about the bounded nature. The current server logic could possible overflow channel in edge case and bring down a heavy loaded worker thread which could be a serious issue.